app/assets/javascripts/components/image_cropper.js

```diff
@@ -0,0 +1,24 @@
+//= require vendor/cropperjs/dist/cropper.js
+
+var $imageCroppers = document.querySelectorAll('[data-module="image-cropper"]')
+
+/* eslint-disable */
+if ($imageCroppers) {
+  $imageCroppers.forEach(function ($imageCropper) {
+    var $image = $imageCropper.querySelector('.app-c-image-cropper__image')
+    if ($image){
+      new Cropper($image,{
+        viewMode: 2,
+        aspectRatio: 3 / 2,
+        autoCrop: true,
+        autoCropArea: .80,
+        guides: false,
+        zoomable: false,
+        highlight: false,
+        minCropBoxWidth: 300,
```

Comment:
Should we synchronise these settings with the minimum dimensions, which appear to be 960x640?

Reply:
Setting the minimum to 960 means you currently won't be able to resize the cropping frame (as it's already constrained in width to 960 by the container), but that can be achieved if I do the extra work to allow the resize of the cropping frame based on the image size (as mentioned in my reply to Kevin)